templates/stanford_person/stanford-person.html.twig

```diff
@@ -7,12 +7,10 @@
     {% endif %}
 
     <div class="content-type__header">
-      <div class="page-title"><h1>{{ first_name }} {{ last_name }}</h1></div>
+      <div class="page-title"><h1>{{ first_name }}</h1></div>
```

Comment:
Dena - did you intend the profile display to be first_name only? Should be {{first_name}} {{last_name}}, right?

Reply:
Works properly with display configuration from ODE-85 (EARTH-1053)